scripts/globals/interaction/interaction_global.lua

```diff
@@ -32,7 +32,7 @@ function InteractionGlobal.loadContainers(shouldReloadRequires)
         package.loaded[interactionContainersPath] = nil
     end
 
-    local containerFiles = require(interactionContainersPath)
+    local containerFiles = GetQuestAndMissionFilenamesList()
```

Comment:
From the CI fail, since a list is being returned, shouldn't the require still be necessary?

Reply:
After some digging, the CI fails were due to some other bits and pieces.